test_1914:
  payload:
    password: 'QiX19+/DsLfV9kXp2ORsP0eTU5Ky9QBWhKg6EkfrupQ='
    reset_access_key: "false" 
test_1915:
  response_msg:
    error_code: 4101
    message_id: "Access denied for account"
    message: "Access denied. Cannot modify another S3 account."   
test_28927:
  response_msg:
    error_code: 12288
    message_id: "EntityAlreadyExists"
    message: 'The request was rejected because it attempted to create an account that already exists.'
test_28928:
  response_msg:
    error_code: 12288
    message_id: "EmailAlreadyExists"
    message: 'The request was rejected because account with this email already exists.'
test_28929:
  response_msg:
    error_code: 12288
    message_id: "AccessKeyAlreadyExists"
    message: 'The request was rejected because account with this access key already exists.'